ansible-scylla-node/tasks/RedHat.yml

```diff
@@ -82,34 +82,36 @@
   when: item.split(".")[-1] == "repo"
   become: true
 
-- name: Install Scylla
+- name: Install {{ scylla_edition }}
   block:
-  - name: Install latest OSS Scylla
+  - name: Install latest {{ scylla_edition }} Scylla
     yum:
-      name: scylla
+      name: "{{ scylla_package_prefix }}"
```

Comment:
I cannot move this to separate patch, since it's connected to overwrite of fetching the machine image package
this will have to be together with machine image package install

Reply:
No, it's not. 
All you do in this hunk is use a variable (that can have either "scylla" or "scylla-enterprise" value) instead of a hard-coded value.

This is a good but generic change that is irrelevant to this patch.
I suggest to take this out into a separate patch.